app/models/service.rb

```diff
@@ -72,13 +72,13 @@ def self.sections
       },
       {
         name: 'Indexes',
-        description: 'Additional data powering ecosystems core services, provided direct for your own use',
+        description: 'Additional data powering ecosyste.ms core services, provided direct for your own use',
```

Comment:
ecosyste.ms is correct brand-wise, but now I'm wondering what's the correct possessive

ecosyste.ms's
ecosyste.m's
ecosyste.ms'